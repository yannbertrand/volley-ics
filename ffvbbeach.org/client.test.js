import { describe, expect, it } from 'vitest'
import getSaison from './client.js'

describe('mockedFfvbbClient', () => {
  it('should return matches list', async () => {
    const saison = '2025/2026'
    const codent = 'PTPL44'
    const poule = 'E1'
    const matches = await getSaison(saison, codent, poule)

    expect(matches).toHaveLength(45)
    expect(matches).toMatchSnapshot()
  })
})
