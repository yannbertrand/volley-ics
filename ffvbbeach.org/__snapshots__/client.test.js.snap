// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`mockedFfvbbClient > should return matches list 1`] = `
[
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-09-22",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0442442",
      "nom": "VLS 7",
    },
    "equipeB": {
      "id": "0440044",
      "nom": "AS LA MONTAGNE 3",
    },
    "heure": "20:00",
    "journee": "01",
    "matchId": "CE1A001",
    "salle": "Gymnase La Martellière, Av. de la Martellière, 44230 Saint-Sébastien-sur-Loire",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-09-25",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440034",
      "nom": "AL MALVILLE 4",
    },
    "equipeB": {
      "id": "0440043",
      "nom": "SVB TREILLIERES 2",
    },
    "heure": "20:00",
    "journee": "01",
    "matchId": "CE1A002",
    "salle": "COMPLEXE ALICE MILLIAT",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-09-22",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0442441",
      "nom": "ORVAULT VB 4",
    },
    "equipeB": {
      "id": "0440018",
      "nom": "USFEN",
    },
    "heure": "19:30",
    "journee": "01",
    "matchId": "CE1A003",
    "salle": "Gymnase de la Cholière, 108 Av. Claude Antoine Peccot, 44700 Orvault",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-09-26",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440051",
      "nom": "VOLLEY BALL LUCEEN 1",
    },
    "equipeB": {
      "id": "0440046",
      "nom": "GRANDCHAMP VB 1",
    },
    "heure": "20:00",
    "journee": "01",
    "matchId": "CE1A004",
    "salle": "HALLE DES SPORTS DE LA MINAIS",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-09-25",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0449539",
      "nom": "VB MESANGER 2",
    },
    "equipeB": {
      "id": "0440037",
      "nom": "VVBL 5",
    },
    "heure": "20:45",
    "journee": "01",
    "matchId": "CE1A005",
    "salle": "Complexe sportif des Cornouailles, 378 Rue Cornouaille, 44522 Mésanger",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-09-30",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440044",
      "nom": "AS LA MONTAGNE 3",
    },
    "equipeB": {
      "id": "0440037",
      "nom": "VVBL 5",
    },
    "heure": "20:15",
    "journee": "02",
    "matchId": "CE1A006",
    "salle": "Salle de sport Jules Ladoumègue, 21 Rue du Drouillard, 44620 La Montagne",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-09-30",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440046",
      "nom": "GRANDCHAMP VB 1",
    },
    "equipeB": {
      "id": "0449539",
      "nom": "VB MESANGER 2",
    },
    "heure": "20:00",
    "journee": "02",
    "matchId": "CE1A007",
    "salle": "Complexe des Cent Sillons, 14 Rue des Cent Sillons, 44119 Grandchamps-des-Fontaines",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-09-29",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440018",
      "nom": "USFEN",
    },
    "equipeB": {
      "id": "0440051",
      "nom": "VOLLEY BALL LUCEEN 1",
    },
    "heure": "20:30",
    "journee": "02",
    "matchId": "CE1A008",
    "salle": "Gymnase Victor Hugo, 29 Rue Paul Bellamy, 44000 Nantes",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-10-03",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440043",
      "nom": "SVB TREILLIERES 2",
    },
    "equipeB": {
      "id": "0442441",
      "nom": "ORVAULT VB 4",
    },
    "heure": "20:00",
    "journee": "02",
    "matchId": "CE1A009",
    "salle": "COMPLEXE DU GESVRE",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-09-29",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0442442",
      "nom": "VLS 7",
    },
    "equipeB": {
      "id": "0440034",
      "nom": "AL MALVILLE 4",
    },
    "heure": "20:00",
    "journee": "02",
    "matchId": "CE1A010",
    "salle": "Gymnase La Martellière, Av. de la Martellière, 44230 Saint-Sébastien-sur-Loire",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-10-09",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440034",
      "nom": "AL MALVILLE 4",
    },
    "equipeB": {
      "id": "0440044",
      "nom": "AS LA MONTAGNE 3",
    },
    "heure": "20:30",
    "journee": "03",
    "matchId": "CE1A011",
    "salle": "COMPLEXE ALICE MILLIAT",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-10-06",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0442441",
      "nom": "ORVAULT VB 4",
    },
    "equipeB": {
      "id": "0442442",
      "nom": "VLS 7",
    },
    "heure": "19:30",
    "journee": "03",
    "matchId": "CE1A012",
    "salle": "Gymnase de la Cholière, 108 Av. Claude Antoine Peccot, 44700 Orvault",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-10-10",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440051",
      "nom": "VOLLEY BALL LUCEEN 1",
    },
    "equipeB": {
      "id": "0440043",
      "nom": "SVB TREILLIERES 2",
    },
    "heure": "20:00",
    "journee": "03",
    "matchId": "CE1A013",
    "salle": "HALLE DES SPORTS DE LA MINAIS",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-10-09",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0449539",
      "nom": "VB MESANGER 2",
    },
    "equipeB": {
      "id": "0440018",
      "nom": "USFEN",
    },
    "heure": "20:45",
    "journee": "03",
    "matchId": "CE1A014",
    "salle": "Complexe sportif des Cornouailles, 378 Rue Cornouaille, 44522 Mésanger",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-10-07",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440037",
      "nom": "VVBL 5",
    },
    "equipeB": {
      "id": "0440046",
      "nom": "GRANDCHAMP VB 1",
    },
    "heure": "20:30",
    "journee": "03",
    "matchId": "CE1A015",
    "salle": "COMPLEXE SEVRE ET MAINE",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-10-14",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440044",
      "nom": "AS LA MONTAGNE 3",
    },
    "equipeB": {
      "id": "0440046",
      "nom": "GRANDCHAMP VB 1",
    },
    "heure": "20:15",
    "journee": "04",
    "matchId": "CE1A016",
    "salle": "Salle de sport Jules Ladoumègue, 21 Rue du Drouillard, 44620 La Montagne",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-10-13",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440018",
      "nom": "USFEN",
    },
    "equipeB": {
      "id": "0440037",
      "nom": "VVBL 5",
    },
    "heure": "20:30",
    "journee": "04",
    "matchId": "CE1A017",
    "salle": "Gymnase Victor Hugo, 29 Rue Paul Bellamy, 44000 Nantes",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-10-17",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440043",
      "nom": "SVB TREILLIERES 2",
    },
    "equipeB": {
      "id": "0449539",
      "nom": "VB MESANGER 2",
    },
    "heure": "20:00",
    "journee": "04",
    "matchId": "CE1A018",
    "salle": "COMPLEXE DU GESVRE",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-10-13",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0442442",
      "nom": "VLS 7",
    },
    "equipeB": {
      "id": "0440051",
      "nom": "VOLLEY BALL LUCEEN 1",
    },
    "heure": "20:00",
    "journee": "04",
    "matchId": "CE1A019",
    "salle": "Gymnase La Martellière, Av. de la Martellière, 44230 Saint-Sébastien-sur-Loire",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-10-16",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440034",
      "nom": "AL MALVILLE 4",
    },
    "equipeB": {
      "id": "0442441",
      "nom": "ORVAULT VB 4",
    },
    "heure": "20:30",
    "journee": "04",
    "matchId": "CE1A020",
    "salle": "COMPLEXE ALICE MILLIAT",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-11-03",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0442441",
      "nom": "ORVAULT VB 4",
    },
    "equipeB": {
      "id": "0440044",
      "nom": "AS LA MONTAGNE 3",
    },
    "heure": "19:30",
    "journee": "05",
    "matchId": "CE1A021",
    "salle": "Gymnase de la Cholière, 108 Av. Claude Antoine Peccot, 44700 Orvault",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-11-07",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440051",
      "nom": "VOLLEY BALL LUCEEN 1",
    },
    "equipeB": {
      "id": "0440034",
      "nom": "AL MALVILLE 4",
    },
    "heure": "20:00",
    "journee": "05",
    "matchId": "CE1A022",
    "salle": "HALLE DES SPORTS DE LA MINAIS",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-11-06",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0449539",
      "nom": "VB MESANGER 2",
    },
    "equipeB": {
      "id": "0442442",
      "nom": "VLS 7",
    },
    "heure": "20:45",
    "journee": "05",
    "matchId": "CE1A023",
    "salle": "Complexe sportif des Cornouailles, 378 Rue Cornouaille, 44522 Mésanger",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-11-04",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440037",
      "nom": "VVBL 5",
    },
    "equipeB": {
      "id": "0440043",
      "nom": "SVB TREILLIERES 2",
    },
    "heure": "20:30",
    "journee": "05",
    "matchId": "CE1A024",
    "salle": "COMPLEXE SEVRE ET MAINE",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-11-04",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440046",
      "nom": "GRANDCHAMP VB 1",
    },
    "equipeB": {
      "id": "0440018",
      "nom": "USFEN",
    },
    "heure": "20:00",
    "journee": "05",
    "matchId": "CE1A025",
    "salle": "Complexe des Cent Sillons, 14 Rue des Cent Sillons, 44119 Grandchamps-des-Fontaines",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-11-18",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440044",
      "nom": "AS LA MONTAGNE 3",
    },
    "equipeB": {
      "id": "0440018",
      "nom": "USFEN",
    },
    "heure": "20:15",
    "journee": "06",
    "matchId": "CE1A026",
    "salle": "Salle de sport Jules Ladoumègue, 21 Rue du Drouillard, 44620 La Montagne",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-11-21",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440043",
      "nom": "SVB TREILLIERES 2",
    },
    "equipeB": {
      "id": "0440046",
      "nom": "GRANDCHAMP VB 1",
    },
    "heure": "20:00",
    "journee": "06",
    "matchId": "CE1A027",
    "salle": "COMPLEXE DU GESVRE",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-11-17",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0442442",
      "nom": "VLS 7",
    },
    "equipeB": {
      "id": "0440037",
      "nom": "VVBL 5",
    },
    "heure": "20:00",
    "journee": "06",
    "matchId": "CE1A028",
    "salle": "Gymnase La Martellière, Av. de la Martellière, 44230 Saint-Sébastien-sur-Loire",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-11-20",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440034",
      "nom": "AL MALVILLE 4",
    },
    "equipeB": {
      "id": "0449539",
      "nom": "VB MESANGER 2",
    },
    "heure": "20:30",
    "journee": "06",
    "matchId": "CE1A029",
    "salle": "COMPLEXE ALICE MILLIAT",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-11-17",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0442441",
      "nom": "ORVAULT VB 4",
    },
    "equipeB": {
      "id": "0440051",
      "nom": "VOLLEY BALL LUCEEN 1",
    },
    "heure": "19:30",
    "journee": "06",
    "matchId": "CE1A030",
    "salle": "Gymnase de la Cholière, 108 Av. Claude Antoine Peccot, 44700 Orvault",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-11-28",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440051",
      "nom": "VOLLEY BALL LUCEEN 1",
    },
    "equipeB": {
      "id": "0440044",
      "nom": "AS LA MONTAGNE 3",
    },
    "heure": "20:00",
    "journee": "07",
    "matchId": "CE1A031",
    "salle": "HALLE DES SPORTS DE LA MINAIS",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-11-26",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0449539",
      "nom": "VB MESANGER 2",
    },
    "equipeB": {
      "id": "0442441",
      "nom": "ORVAULT VB 4",
    },
    "heure": "20:45",
    "journee": "07",
    "matchId": "CE1A032",
    "salle": "Complexe sportif des Cornouailles, 378 Rue Cornouaille, 44522 Mésanger",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-11-25",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440037",
      "nom": "VVBL 5",
    },
    "equipeB": {
      "id": "0440034",
      "nom": "AL MALVILLE 4",
    },
    "heure": "20:30",
    "journee": "07",
    "matchId": "CE1A033",
    "salle": "COMPLEXE SEVRE ET MAINE",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-11-25",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440046",
      "nom": "GRANDCHAMP VB 1",
    },
    "equipeB": {
      "id": "0442442",
      "nom": "VLS 7",
    },
    "heure": "20:00",
    "journee": "07",
    "matchId": "CE1A034",
    "salle": "Complexe des Cent Sillons, 14 Rue des Cent Sillons, 44119 Grandchamps-des-Fontaines",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-11-24",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440018",
      "nom": "USFEN",
    },
    "equipeB": {
      "id": "0440043",
      "nom": "SVB TREILLIERES 2",
    },
    "heure": "20:30",
    "journee": "07",
    "matchId": "CE1A035",
    "salle": "Gymnase Victor Hugo, 29 Rue Paul Bellamy, 44000 Nantes",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-12-02",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440044",
      "nom": "AS LA MONTAGNE 3",
    },
    "equipeB": {
      "id": "0440043",
      "nom": "SVB TREILLIERES 2",
    },
    "heure": "20:15",
    "journee": "08",
    "matchId": "CE1A036",
    "salle": "Salle de sport Jules Ladoumègue, 21 Rue du Drouillard, 44620 La Montagne",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-12-01",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0442442",
      "nom": "VLS 7",
    },
    "equipeB": {
      "id": "0440018",
      "nom": "USFEN",
    },
    "heure": "20:00",
    "journee": "08",
    "matchId": "CE1A037",
    "salle": "Gymnase La Martellière, Av. de la Martellière, 44230 Saint-Sébastien-sur-Loire",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-12-04",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440034",
      "nom": "AL MALVILLE 4",
    },
    "equipeB": {
      "id": "0440046",
      "nom": "GRANDCHAMP VB 1",
    },
    "heure": "20:30",
    "journee": "08",
    "matchId": "CE1A038",
    "salle": "COMPLEXE ALICE MILLIAT",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-12-01",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0442441",
      "nom": "ORVAULT VB 4",
    },
    "equipeB": {
      "id": "0440037",
      "nom": "VVBL 5",
    },
    "heure": "19:30",
    "journee": "08",
    "matchId": "CE1A039",
    "salle": "Gymnase de la Cholière, 108 Av. Claude Antoine Peccot, 44700 Orvault",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-12-05",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440051",
      "nom": "VOLLEY BALL LUCEEN 1",
    },
    "equipeB": {
      "id": "0449539",
      "nom": "VB MESANGER 2",
    },
    "heure": "20:00",
    "journee": "08",
    "matchId": "CE1A040",
    "salle": "HALLE DES SPORTS DE LA MINAIS",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-12-10",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0449539",
      "nom": "VB MESANGER 2",
    },
    "equipeB": {
      "id": "0440044",
      "nom": "AS LA MONTAGNE 3",
    },
    "heure": "20:45",
    "journee": "09",
    "matchId": "CE1A041",
    "salle": "Complexe sportif des Cornouailles, 378 Rue Cornouaille, 44522 Mésanger",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-12-09",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440037",
      "nom": "VVBL 5",
    },
    "equipeB": {
      "id": "0440051",
      "nom": "VOLLEY BALL LUCEEN 1",
    },
    "heure": "20:30",
    "journee": "09",
    "matchId": "CE1A042",
    "salle": "COMPLEXE SEVRE ET MAINE",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-12-09",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440046",
      "nom": "GRANDCHAMP VB 1",
    },
    "equipeB": {
      "id": "0442441",
      "nom": "ORVAULT VB 4",
    },
    "heure": "20:00",
    "journee": "09",
    "matchId": "CE1A043",
    "salle": "Complexe des Cent Sillons, 14 Rue des Cent Sillons, 44119 Grandchamps-des-Fontaines",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-12-08",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440018",
      "nom": "USFEN",
    },
    "equipeB": {
      "id": "0440034",
      "nom": "AL MALVILLE 4",
    },
    "heure": "20:30",
    "journee": "09",
    "matchId": "CE1A044",
    "salle": "Gymnase Victor Hugo, 29 Rue Paul Bellamy, 44000 Nantes",
    "score": "",
    "set": "",
    "total": "",
  },
  {
    "arbitres": [
      "",
      "",
    ],
    "date": "2025-12-12",
    "entite": "PTPL44",
    "equipeA": {
      "id": "0440043",
      "nom": "SVB TREILLIERES 2",
    },
    "equipeB": {
      "id": "0442442",
      "nom": "VLS 7",
    },
    "heure": "20:00",
    "journee": "09",
    "matchId": "CE1A045",
    "salle": "COMPLEXE DU GESVRE",
    "score": "",
    "set": "",
    "total": "",
  },
]
`;
